### PR TITLE
CN 4271 loan application success modal

### PR DESCRIPTION
**Pull request by srkela**
PR Checklist:
- [x] I have named the PR correctly
- [x] I have added at least two reviewers
- [x] I have assigned the PR to myself
- [x] I have merged develop into my feature branch and resolved conflicts
- [x] I have tested all new features locally
- [x] I have tested relevant flow locally

### How to name a PR
- If it includes only one issue [CN-1234 Issue name]
- If it includes many issues [CN-1234, CN-2345, CN-3456 What was updated/changed/added]
- If the PR is not ready yet, add [DRAFT] in front of the name

### How to test locally
1. Test happy path
2. Test other paths
3. Remove all warnings

### How to relevant flow locally
- If working on part of CelPay, check the whole happy path

### CODE REVIEW

File: app/components/modals/LoanApplicationSuccessModal/LoanApplicationSuccessModal.js [hunk 1]
```diff
@@ -0,0 +1,114 @@
...
+      reason: LOAN_PAYMENT_REASONS.INTEREST_PREPAYMENT,
+    });
+    actions.closeModal();
+  };
+
+  render() {
+    const { actions, formData } = this.props;
+    const style = LoanApplicationSuccessModalStyle();
+
+    if (formData.termOfLoan < 12)
```
Comment (fjovakaric) [inline app/components/modals/LoanApplicationSuccessModal/LoanApplicationSuccessModal.js]: Check `loanActions.js`

```
      if (
        Number(formData.loanAmount) <= Number(automaticLoanLimit) &&
        formData.coin !== "USD"
      ) {
        dispatch(openModal(MODALS.LOAN_APPLICATION_SUCCESS_MODAL));
      }
```
Condition is for the loan amount, not term of loan

File: app/components/modals/LoanApplicationSuccessModal/LoanApplicationSuccessModal.js [hunk 2]
```diff
@@ -0,0 +1,114 @@
...
+  }),
+  dispatch => ({ actions: bindActionCreators(appActions, dispatch) })
+)
+class LoanApplicationSuccessModal extends Component {
+  static propTypes = {
+    loanId: PropTypes.number.isRequired,
+  };
+  static defaultProps = {};
+
+  closeModalHandler = () => {
```
Comment (fjovakaric) [inline app/components/modals/LoanApplicationSuccessModal/LoanApplicationSuccessModal.js]: rename method to something for prepayment

File: app/components/modals/LoanApplicationSuccessModal/LoanApplicationSuccessModal.js (hunk 1 above)
Comment (srkela) [reply app/components/modals/LoanApplicationSuccessModal/LoanApplicationSuccessModal.js]: Fixed

File: app/components/modals/LoanApplicationSuccessModal/LoanApplicationSuccessModal.js (hunk 2 above)
Comment (srkela) [reply app/components/modals/LoanApplicationSuccessModal/LoanApplicationSuccessModal.js]: Done

**Pull request merged**